src/components/Editor.vue

```diff
@@ -77,6 +77,11 @@
 				:is-rich-editor="isRichEditor" />
 		</Wrapper>
 		<Assistant v-if="$editor" />
+		<Translate :show.sync="translateModal"
```

Comment:
In order for `:show.sync` to properly work you'd need to emit `update:show` from the `Translate` modal. I think that does not happen yet. Once you have that you don't need to handle `@close` with `hideTranslate` below anymore as the `translateModal` state would be synced already via `:show.sync` already.